documentation/library_guide/introduction.rst

```diff
@@ -98,6 +98,7 @@ Known Limitations
   vector execution policies when building a program with GCC 10 and using ``-O0`` option.
 * Compiling ``reduce`` and ``transform_reduce`` algorithms with the Intel DPC++ Compiler, versions 2021 and older,
   may result in a runtime error. To fix this issue, use an Intel DPC++ Compiler version 2022 or newer.
+* Compilation on Windows requires the addition of the option ``/EHsc`` to the compile line in order to avoid errors about oneDPL's experimental ranges support throwing exceptions.
```

Comment:
My suggestion:

When you compile on Windows, add the option ``/EHsc`` to the compile line to avoid oneDPL errors when experimental ranges support throwing exceptions.

Reply:
LGTM